components/oauth/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -142,7 +142,7 @@ public static String getUserInfoClaimDialect() {
     public static String[] extractCredentialsFromAuthzHeader(String authorizationHeader)
             throws OAuthClientException {
         String[] splitValues = authorizationHeader.trim().split(" ");
-        if(splitValues.length == 2) {
+        if(splitValues.length >= 2) {
```

Comment:
This is fixed in a later commit.since the PR is closed the updated version is not visible here.